coldfront/overlays/ocp-staging/patches/deployments/coldfront-deployment.yaml

```diff
@@ -0,0 +1,93 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: coldfront-deployment
+spec:
+  selector:
```

Comment:
This selector element is provided by the base `kustomization.yaml` and does not need to be provided in the patch.